utils/cq/daos_pylint.py

```diff
@@ -433,8 +433,8 @@ def msg_to_github(vals):
             if msg.symbol in ('condition-evals-to-constant'):
                 promote_to_error = True
 
-            # All non-scons code should be clean now.
-            if scons:
+            # All scons and ftest code should be clean now.
+            if scons or ftest:
```

Comment:
This isn't technically correct now but we can fix it up later when we remove no-longer-needed code here.